src/assets/stylesheets/theme/fonts.scss

```diff
@@ -7,9 +7,7 @@
   font-display: swap;
   font-style: normal;
   font-weight: 300;
-  src:
-    local(''),
-    url('../fonts/FiraSans-Light-300.woff2') format('woff2');
+  src: url('../fonts/FiraSans-Light-300.woff2') format('woff2');
```

Comment:
* [ ] FYI as a next step we could move the two initial fonts from the app assets to the content and set the CSS default to a web safe font.